src/io/plot.jl

```diff
@@ -1,10 +1,13 @@
-export DiGraph, plot
+export 
+    save_as_dot, 
+    save_as_tex, 
```

Comment:
I think `save_as_tex` was in another file and was removed? Also don't see `save_as_dot` and `save_as_dot2tex` in this file.